graphlearn/python/config.py

```diff
@@ -106,4 +106,10 @@ def set_neg_sampler_retry_times(times):
   pywrap.set_neg_sampler_retry_times(times)
 
 def set_field_delimiter(delimiter="\t"):
-  pywrap.set_field_delimiter(delimiter)
\ No newline at end of file
+  pywrap.set_field_delimiter(delimiter)
+
+def set_vineyard_graph_id(graph_id):
```

Comment:
Better provide an example of the python side of vineyard?

Reply:
The python changes will be pushed in another PR (see overall plan in https://github.com/alibaba/graph-learn/pull/208#issue-1310757720).